bindings/azure/blobstorage/blobstorage.go

```diff
@@ -251,32 +254,32 @@ func (a *AzureBlobStorage) get(req *bindings.InvokeRequest) (*bindings.InvokeRes
 	}
 
 	ctx := context.TODO()
-	resp, err := blobURL.Download(ctx, 0, azblob.CountToEnd, azblob.BlobAccessConditions{}, false)
+	offset, err := strconv.ParseInt(req.Metadata[metadataKeyOffset],10,64)
+	if err != nil {
+		offset = 0
+	}
+	count, err := strconv.ParseInt(req.Metadata[metadataKeyCount],10,64)
+	if err != nil {
+		count = 0
+	}
+	resp, err := blobURL.Download(ctx, offset, count, azblob.BlobAccessConditions{}, false)
```

Comment:
There is no need to add addittional checks because .Download method will return a value that is not nil for the err variable in case that something goes wrong and we will return an error after that.